### ISSUE TITLE
YA Regex that breaks syntax highlighting

### ISSUE THREAD

**Issue by lipkau** [PowerShell/EditorSyntax#75]
### Description

I have the following line of code in my script:

```powershell
[ValidatePattern('(?=^.{1,254}$)(^(?:(?!\d+\.)[a-zA-Z0-9_\-]{1,63}\.?)+(?:[a-zA-Z]{2,})$)')]
```

Here is how it looks in VSCode:

![image](https://user-images.githubusercontent.com/4992870/38553296-18e26da2-3cbf-11e8-8fcf-c841423761fe.png)

### Environment

```powershell
PS C:\> code -v
1.22.1
950b8b0d37a9b7061b6f0d291837ccc4015f5ecdx64

PS C:\> $pseditor.EditorServicesVersion

Major  Minor  Build  Revision
-----  -----  -----  --------
1      6      0      0

PS C:\> code --list-extensions --show-versions
[...]
ms-vscode.PowerShell@1.6.0
[...]

PS C:\> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      6.0.1
PSEdition                      Core
GitCommitId                    v6.0.1
OS                             Microsoft Windows 10.0.14393
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by omniomi** [comment]
I managed to find the issue. Need to do some more testing to make sure the fix doesn't introduce new problems and then I will submit the PR.

![regex-fix](https://user-images.githubusercontent.com/12937335/39559099-fe105798-4e50-11e8-9160-bec840267c92.png)

**Comment by tylerl0706** [comment]
Niceee 👍

**Comment by omniomi** [comment]
Fixed by #94
